nomad/nomad_node_configs/initial_server.hcl

```diff
@@ -4,6 +4,12 @@ bind_addr = "0.0.0.0"
 server {
   enabled = true
   bootstrap_expect = 1
+  preemption_config {
```

Comment:
https://developer.hashicorp.com/nomad/docs/configuration/server#bootstrapping-with-a-custom-scheduler-config

Does the preemption_config need to be in the default_scheduler_config stanza, or does this work?

Reply:
You are right... I had missed that. Corrected now.